.header.md

```diff
@@ -41,7 +41,7 @@ module "vpc" {
 
 ## Reserved Subnet Key Names
 
-There are 2 reserved keys for subnet key names in var.subnets corresponding to types "public" and "transit_gateway". Other custom subnet key names are valid are and those subnets will be private subnets.
+There are 3 reserved keys for subnet key names in var.subnets corresponding to types "public", "transit_gateway", and "core_network". Other custom subnet key names are valid are and those subnets will be private subnets.
```

Comment:
```suggestion
There are 3 reserved keys for subnet key names in var.subnets corresponding to types "public", "transit_gateway", and "core_network" [(a NetworkManager CloudWAN feature)](https://docs.aws.amazon.com/vpc/latest/cloudwan/cloudwan-networks-working-with.html). Other custom subnet key names are valid are and those subnets will be private subnets.
```

Reply:
my thought here is that _most people_ dont know cloudwan but the term "core network" feels very confusing.